### PR TITLE
Handle when the file is not in the correct format

### PR DESCRIPTION
**Pull request by gearedmonkey**
### Summary

Handle when the excel file is not in the correct format.

### Checklist

- [x] Have you added unit and functional tests as appropriate?
- [x] Did you update/add documentation for new methods or changed functionality?
- [x] Have you merged the target branch down into this one?

### How to Review

1. Review code?
2. Follow steps in other PR below
3. Approve to your consent.

### Links

- TheFourFifths/consus-client/pull/75

### CODE REVIEW

File: src/api/student.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ app.post('/', (req, res) => {
                 email: s['E-mail'],
                 major: s['Major']
             };
+            if (student.name === undefined || student.name === undefined || student.status === undefined ||
```
Comment (mattmahn) [inline src/api/student.js]: `if(Object.values(student).includes(undefined))` if you're into that sort of thing.

File: src/api/student.js [hunk 2]
```diff
@@ -29,6 +29,10 @@ app.post('/', (req, res) => {
...
+                    student.email === undefined || student.major === undefined) {
+                return res.failureJson('File not in the correct format!');
```
Comment (mattmahn) [inline src/api/student.js]: NACK: please set the [status code](http://expressjs.com/en/4x/api.html#res.status) to something descriptive, like 400 Bad Request.

Comment (gearedmonkey) [inline src/api/student.js]: Please read [api.js](https://github.com/TheFourFifths/consus/blob/dev/src/lib/api.js#L23) >.<

**Comment by gearedmonkey** [comment]
@mattmahn changed.

File: src/api/student.js (hunk 1 above)
Comment (thofmann) [inline src/api/student.js]: >`student.name === undefined || student.name === undefined`

If x or x?

Comment (gearedmonkey) [inline src/api/student.js]: @thofmann ha, that's fixed.

**Pull request merged**